sdk/confidentialledger/test-resources.json

```diff
@@ -31,75 +31,31 @@
       }
     }
   },
-  "variables": {
-    "mgmtApiVersion": "2019-04-01",
-    "location": "[resourceGroup().location]",
-    "primaryAccountName": "[parameters('baseName')]",
-    "encryption": {
-      "services": {
-        "file": {
-          "enabled": true
-        },
-        "blob": {
-          "enabled": true
-        }
-      },
-      "keySource": "Microsoft.Storage"
-    },
-    "networkAcls": {
-      "bypass": "AzureServices",
-      "virtualNetworkRules": [],
-      "ipRules": [],
-      "defaultAction": "Allow"
-    }
-  },
   "resources": [
-    {
-      "type": "Microsoft.Storage/storageAccounts",
-      "apiVersion": "[variables('mgmtApiVersion')]",
-      "name": "[variables('primaryAccountName')]",
-      "location": "[variables('location')]",
-      "sku": {
-        "name": "Standard_RAGRS",
-        "tier": "Standard"
-      },
-      "kind": "StorageV2",
-      "properties": {
-        "networkAcls": "[variables('networkAcls')]",
-        "supportsHttpsTrafficOnly": true,
-        "encryption": "[variables('encryption')]",
-        "accessTier": "Hot"
-      }
-    },
     {
       "type": "Microsoft.ConfidentialLedger/ledgers",
       "apiVersion": "2020-12-01-preview",
       "name": "[parameters('baseName')]",
-      "location": "[parameters('location')]",
+      "location": "EASTUS",
```

Comment:
Do we want to hardcode the location?

Reply:
For now, yes because there are region restrictions for the service. We should be able to accept the default eventually.